components/src/core/components/Input/_variables.scss

```diff
@@ -89,6 +89,7 @@ $oxd-dropdown-option-color--disabled: $oxd-interface-gray-lighten-1-color !defau
 $oxd-dropdown-option-color--selected: $oxd-interface-gray-lighten-1-color !default;
 $oxd-dropdown-option-background: transparent !default;
 $oxd-dropdown-option-background--focused: #f6f7f9 !default;
+$oxd-dropdown-option-background--selected: $oxd-interface-gray-lighten-2-color !default;
```

Comment:
The sorting dropdown doesn't have the change. Check all the places and apply the fix

Reply:
@amilaw I checked the code base but there's no optioin to show selected sorting option in the sort dropdown therefore I didn't have to set the background color.
But in the calendar component there was a selected background color indication. and updated that as well. which is mentioned [here](https://orangehrmenterprise.atlassian.net/browse/RMP-576?focusedCommentId=16592)

Reply:
Please change the sorting color as discussed

Reply:
@amilaw @lgtkaushalya I have observed this issue and what we saw on last meeting was not the background colour of the selected sorting item. That was the hover and the focus state and there's no option to show the previous selected item from the table sorting dropdown.
This is a separate implementation and not related with the dropdown options in the other areas such as select dropdown